sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -1002,6 +1006,163 @@ mod test_topology_manager_policy {
 
 // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+/// imageGCHighThresholdPercent is the percent of disk usage after which image
+/// garbage collection is always run. The percent is calculated by dividing this
+/// field value by 100, so this field must be between 0 and 100, inclusive. When
+/// specified, the value must be greater than imageGCLowThresholdPercent.
+/// Default: 85
+/// https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct ImageGCHighThresholdPercent {
+    // Important note: custom types currently need to have `string_impls_for!()`
+    // after their TryFrom<&str> implementation. (this requires that the inner
+    // type of a custom Kubernetes settings type is a String).
+    // Ideally, `string_impls_for!()` would be accompanied by `i32_impls_for!()`
+    // and macros for other `inner` types.
```

Comment:
I don't know if this comment is relevant since there is a GH issue for the problem described. Maybe just add the link for the issue here and remove it when the problem is fixed?

Reply:
Oops, meant to remove this in my latest change. Thanks!